core/state/snapshot/disklayer.go

```diff
@@ -107,17 +109,63 @@ func (dl *diskLayer) AccountRLP(hash common.Hash) ([]byte, error) {
 		return nil, ErrNotCoveredYet
 	}
 	// If we're in the disk layer, all diff layers missed
+	routeid := cachemetrics.Goid()
+	isSyncMainProcess := cachemetrics.IsSyncMainRoutineID(routeid)
+	isMinerMainProcess := cachemetrics.IsMinerMainRoutineID(routeid)
 	snapshotDirtyAccountMissMeter.Mark(1)
 
+	hitInL3 := false
+	hitInDisk := false
+	startGetInDisk := time.Now()
+	defer func() {
+		// if mainProcess
+		if isSyncMainProcess {
+			syncL2AccountMissMeter.Mark(1)
+			if hitInL3 {
+				syncL3AccountHitMeter.Mark(1)
+				cachemetrics.RecordCacheDepth("CACHE_L3_ACCOUNT")
+				cachemetrics.RecordCacheMetrics("CACHE_L3_ACCOUNT", start)
+				cachemetrics.RecordTotalCosts("CACHE_L3_ACCOUNT", start)
+			}
+			if hitInDisk {
+				syncL3AccountMissMeter.Mark(1)
+				cachemetrics.RecordCacheDepth("DISK_L4_ACCOUNT")
+				cachemetrics.RecordCacheMetrics("DISK_L4_ACCOUNT", startGetInDisk)
```

Comment:
Little difference between`startGetInDisk` and `start`? Why use different ones here>